mmr-backend/book.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+from enum import Enum
+
+
+class Book:
+    def __init__(self, id: str, title: str, synopsis: str, author: str, genres: list[Book.Genre], publisher: str, publishing_date: str):
```

Comment:
Edition is missing.
Also, this should probably be a dataclass. Data on a book does not change.